proxmoxtf/resource/container/container.go

```diff
@@ -52,7 +52,10 @@ const (
 	dvCPUUnits                          = 1024
 	dvDescription                       = ""
 	dvDevicePassthroughMode             = "0660"
+	dvDiskACL                           = false
 	dvDiskDatastoreID                   = "local"
+	dvDiskQuota                         = false
+	dvDiskReplicate                     = false
```

Comment:
this has to be `false` despite PVE default is true. We can't really heave "true by default" boolean attributes in SDK-based resources, unfortunately.